app/Models/Sensor.php

```diff
@@ -197,6 +198,26 @@ public function syncGroup(): string
         return "$this->sensor_class-$this->poller_type";
     }
 
+    /**
+     * @param  Builder  $query
+     * @return Builder
+     */
+    public function scopeIsCritical($query)
+    {
+        return $query->whereColumn([
```

Comment:
Try: `return $query->whereColumn('sensor_current','<','sensor_limit_low')->orWhereColumn('sensor_current','>','sensor_limit')`

Reply:
The scopeIsXxx is mimicked from existing functionality.